packages/cdk/lib/transcription-service.ts

```diff
@@ -462,6 +462,24 @@ export class TranscriptionService extends GuStack {
 		// allow worker to write messages to the dead letter queue
 		transcriptionDeadLetterQueue.grantSendMessages(transcriptionWorkerASG);
 
+		// SQS queue for media download tasks from API lambda to media-downloader service
+		const mediaDownloadTaskQueue = new Queue(
+			this,
+			`${APP_NAME}-media-download-task-queue`,
+			{
+				fifo: true,
+				queueName: `${APP_NAME}-media-download-task-queue-${this.stage}.fifo`,
+				visibilityTimeout: Duration.seconds(30),
+				contentBasedDeduplication: true,
+				deadLetterQueue: {
+					queue: transcriptionDeadLetterQueue,
```

Comment:
Do we want transcription and download to share a dead letter queue? It seems like debugging might be easier if they're separate.

Reply:
eek good spot! This was a copy paste failure, I agree they should be separate